modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl

```diff
@@ -225,7 +225,7 @@
 								/>
 
 								<#list restArticle.contentFields as fieldData>
-									<#if fieldData.contentFieldValue.data?has_content>
+									<#if fieldData.contentFieldValue.data?has_content && validator.isNotNull(fieldData.contentFieldValue.data)>
```

Comment:
do we need both the `has_content` check and the `isNotNull` check? would just the `isNotNull` be sufficient since that would check for empty content as well?

Reply:
Just to add... based on what Jair had mentioned before, can we avoid using just `?has_content` and use Boolean()? I'm not sure if ftl would support this though which is why I'm asking. I had updated the frontend code to keep it consistent as I agreed with Jair that we should use Boolean() to be clear when we're validating or if statement check.

Reply:
@kylembischof I just tested with `validator.isNotNull(fieldData.contentFieldValue.data)` and it's working as we want.

Reply:
That's fine, thanks for checking. Can just stick with the isNotNull.